app/assets/javascripts/discourse/app/components/post-list/index.gjs

```diff
@@ -68,13 +71,27 @@ export default class PostList extends Component {
     {{/if}}
 
     <LoadMore @selector=".post-list-item" @action={{this.loadMore}}>
-      <div class="post-list">
+      <div class="post-list" ...attributes>
         {{#each @posts as |post|}}
           <PostListItem
             @post={{post}}
             @additionalItemClasses={{@additionalItemClasses}}
             @titleAriaLabel={{@titleAriaLabel}}
-          />
+            @showUserInfo={{@showUserInfo}}
+          >
+            <:abovePostItemHeader>
+              {{yield post to="abovePostItemHeader"}}
+            </:abovePostItemHeader>
+            <:belowPostItemMetaData>
+              {{yield post to="belowPostItemMetaData"}}
+            </:belowPostItemMetaData>
+            <:abovePostItemExcerpt>
+              {{yield post to="abovePostItemExcerpt"}}
+            </:abovePostItemExcerpt>
+            <:belowPostItem>
+              {{yield post to="belowPostItem"}}
+            </:belowPostItem>
```

Comment:
This is a cool pattern for re-usability, while still allowing differences for each use case 😍